src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.cpp

```diff
@@ -337,7 +345,15 @@ void WebRTCTransportProviderServer::HandleSolicitOffer(HandlerContext & ctx, con
 
     // Prepare the arguments for the delegate.
     Delegate::OfferRequestArgs args;
-    args.sessionId             = GenerateSessionId();
+    uint16_t sessionId;
+    CHIP_ERROR err = GenerateSessionId(sessionId);
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(Zcl, "HandleSolicitOffer: Cannot generate session ID");
```

Comment:
we should log the error here just in case we later have more errors (in debug at least we will see original file:line).

Same for below. Also we should set status with the received error rather than trying to say "any error is resource exhausted".